src/views/ItemHome.vue

```diff
@@ -64,17 +70,21 @@
 </template>
 
 <script>
-import { mapGetters, mapState } from 'vuex';
+import { mapGetters, mapState, mapActions } from 'vuex';
+import { isBefore, addDays } from 'date-fns';
+import tabIdsHelper from '@/util/tabUtils';
+import { periodDates } from '@/util';
 import ContentLoaderItem from '@/components/ContentLoader/ContentLoaderItem.vue';
 import ContentLoaderActionBar from '@/components/ContentLoader/ContentLoaderActionBar.vue';
 import WidgetsMobile from '@/components/widgets/WidgetsMobile.vue';
+import TabList from '@/components/TabList.vue';
+import TabPanel from '@/components/TabPanel.vue';
 
 export default {
   name: 'ItemHome',
 
   components: {
     WidgetsMobile,
```

Comment:
Not sure what causes it directly, but `WidgetMobile` now raises an error on `activePeriod` beeing `null`: Do we have to be load the component asynchronously, or add some additional checks? 🤔